dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -38,9 +38,16 @@ class LeapHybridSampler(dimod.Sampler):
 
     Uses parameters set in a configuration file, as environment variables, or
     explicitly as input arguments for selecting and communicating with a hybrid
-    solver. For more information, see
+    solver, as described in
     `D-Wave Cloud Client <https://docs.ocean.dwavesys.com/en/stable/docs_cloud/sdk_index.html>`_.
 
+    :term:`Solver` selection---handled by :ref:`dwave-cloud-client <sdk_index_cloud>`'s
+    :meth:`~dwave.cloud.client.Client.get_solvers` method---defaults to filtering
+    solvers you have access to by
+    `solver properties <https://docs.dwavesys.com/docs/latest/c_solver_3.html>`_
+    ``category=hybrid`` and ``supported_problem_type=bqm`` and returns online
+    solvers ordered by latest ``version``.
```

Comment:
Worth mentioning solver has to be overridden in the code. Config/env definition will be ignored. Open issue https://github.com/dwavesystems/dwave-system/issues/363.

Reply:
Are you asking that we document a broken behavior? The documentation should describe the correct behavior, and then the Issues describe any problems. I assume we'll fix #363.

Reply:
Yes, I think we should document any gotchas/quirks. The current behavior is intended, but unexpected for users coming from DWaveSampler.